inc/editor-panel.php

```
@@ -0,0 +1,280 @@
+		);
+	}
+
+	// Pass i18n strings and field options to JavaScript.
+	// i18n文字列とフィールドオプションをJavaScriptに渡す。
+	wp_localize_script(
+		'vgjpm-editor-panel',
+		'vgjpmPanelData',
+		array(
+			'i18n'       => array(
+				// Panel titles / パネルタイトル
+				'jobInfo'             => __( 'Job Information', 'vk-google-job-posting-manager' ),
```

PHPStan (2.1.46)

[error] Function __ not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)